doc/mkdocs/docs/api/macros/json_has_cpp_11.md

```diff
@@ -0,0 +1,28 @@
+# JSON_HAS_CPP_11, JSON_HAS_CPP_14, JSON_HAS_CPP_17, JSON_HAS_CPP_20
+
+```cpp
+#define JSON_HAS_CPP_11
+#define JSON_HAS_CPP_14
+#define JSON_HAS_CPP_17
+#define JSON_HAS_CPP_20
+```
+
+The library targets C++11, but also supports some features introduced in later C++ versions (e.g., `std::string_view`
+support for C++17). For these new features, the library implements some preprocessor checks to determine the C++
+standard. By defining any of these symbols, the internal check is overridden and the provided C++ version is
+unconditionally assumed. This can be helpful for compilers that only implement parts of the standard and would be
+detected incorrectly.
+
+## Default definition
+
+The default value is detected based on the preprocessor macros `#!cpp __cplusplus`, `#!cpp _HAS_CXX17`, or
+`#!cpp _MSVC_LANG`.
```

Comment:
And `_HAS_CXX14`. The list should stop growing now, as even MS is starting to embrace `__cplusplus`.